### DISCUSSION TITLE
Issue when attempting to run yarn install in code-server

### DISCUSSION THREAD

**Discussion by CaptMorg70** [coder/code-server#5766] [Q&A]
Hello,

I am attempting to follow the Development workflow in the Contributing.md file.

When I attempt to run the yarn install, I am getting the error:

$ ./ci/dev/postinstall.sh 
: invalid option nameset: pipefail

Has anyone ran into this issue or know a way to get this to work?

Running in Powershell using WSL.

**Comment by jsjoeio** [comment]
Hmm 🤔 I develop on macOS and @code-asher on Linux so I don't know if we've tried on WSL. Maybe it's not compatible with Powershell? Can you try with bash?

**Comment by CaptMorg70** [comment]
Sorry. It is a imgur link. Having an issue attaching the picture directly to the discussion.

https://imgur.com/TrbW1g7

This is running bash. Ubuntu is the OS. It seems to be having the same issue. I class mate of mine is on macOS and has not had any issues getting yarn to run.

**Reply by jsjoeio** [reply]
Hmm...I'm not sure. I found [this](https://www.reddit.com/r/windows/comments/8443od/bash_script_invalid_option_nameset_pipefail/) but no solution. Any ideas @code-asher ?

**Reply by CaptMorg70** [reply]
I found a similar resource and could get the pipe fail command to run like that link.

I’ve tried running directly the postinstall.sh script in the /ci/dev folder and got the same error. This was with bash and with wsl.

So I think it could be an issue with the script in windows? That’s my only guess atm

**Reply by jsjoeio** [reply]
Interesting...

> So I think it could be an issue with the script in windows? That’s my only guess atm

Yeah, I think you're right 🤔 Hoping when Asher sees this he might have some insight

**Reply by code-asher** [reply]
I am not very familiar with WSL but I tried installing it on a Windows machine and upon reboot it opened a shell where I was able to clone and install code-server without any issues.

My guess is it has to do with Powershell.  For me there is an `Ubuntu` app that it launches.
